app/brave_main_delegate.cc

```diff
@@ -155,7 +155,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   // Disabled features.
   const std::unordered_set<const char*> disabled_features = {
       autofill::features::kAutofillServerCommunication.name,
-      network::features::kNetworkService.name,
+      // network::features::kNetworkService.name,
```

Comment:
Temporary to run tests

Reply:
so you'll switch this back before merging?

Reply:
yes

Reply:
switched back